examples/05-Q3D/Q2D_Example_Stripline.py

```diff
@@ -0,0 +1,190 @@
+top_line_obj = q.modeler.create_polyline([[0, layer_2_uh, 0], [sig_top_w, layer_2_uh, 0]])
+q.modeler.move([top_line_obj], [delta_w_half, 0, 0])
+q.modeler.connect([base_line_obj, top_line_obj])
+q.modeler.move([base_line_obj], ["{}+{}".format(co_gnd_w, clearance), 0, 0])
+
+# Create negative signal
+base_line_obj = q.modeler.create_polyline([[0, layer_2_lh, 0], [sig_w, layer_2_lh, 0]], name="signal_n")
+top_line_obj = q.modeler.create_polyline([[0, layer_2_uh, 0], [sig_top_w, layer_2_uh, 0]])
+q.modeler.move([top_line_obj], [delta_w_half, 0, 0])
+q.modeler.connect([base_line_obj, top_line_obj])
+q.modeler.move([base_line_obj], ["{}+{}+{}+{}".format(co_gnd_w, clearance, sig_w, sig_gap), 0, 0])
+
+###############################################################################
+# Create coplanar ground
+base_line_obj = q.modeler.create_polyline([[0, layer_2_lh, 0], [co_gnd_w, layer_2_lh, 0]], name="co_gnd_left")
+top_line_obj = q.modeler.create_polyline([[0, layer_2_uh, 0], [co_gnd_top_w, layer_2_uh, 0]])
+q.modeler.move([top_line_obj], [delta_w_half, 0, 0])
+q.modeler.connect([base_line_obj, top_line_obj])
+
+base_line_obj = q.modeler.create_polyline([[0, layer_2_lh, 0], [co_gnd_w, layer_2_lh, 0]], name="co_gnd_right")
+top_line_obj = q.modeler.create_polyline([[0, layer_2_uh, 0], [co_gnd_top_w, layer_2_uh, 0]])
+q.modeler.move([top_line_obj], [delta_w_half, 0, 0])
+q.modeler.connect([base_line_obj, top_line_obj])
+q.modeler.move([base_line_obj], ["{}+{}*2+{}*2+{}".format(co_gnd_w, clearance, sig_w, sig_gap), 0, 0])
+
+###############################################################################
+# Create reference ground plane
+q.modeler.create_rectangle(position=[0, layer_1_lh, 0], dimension_list=[model_w, cond_h], name="ref_gnd_u")
+q.modeler.create_rectangle(position=[0, layer_3_lh, 0], dimension_list=[model_w, cond_h], name="ref_gnd_l")
+
+###############################################################################
+# Create dielectric
+q.modeler.create_rectangle(
+    position=[0, layer_1_uh, 0], dimension_list=[model_w, core_h], name="Core", matname="FR4_epoxy"
+)
+q.modeler.create_rectangle(
+    position=[0, layer_2_uh, 0], dimension_list=[model_w, pp_h], name="Prepreg", matname="FR4_epoxy"
+)
+q.modeler.create_rectangle(
+    position=[0, layer_2_lh, 0], dimension_list=[model_w, cond_h], name="Filling", matname="FR4_epoxy"
+)
+
+###############################################################################
+# Assign conductors
+###############################################################################
+# Signal
+
+obj = q.modeler.get_object_from_name("signal_p")
+q.assign_single_conductor(
+    name=obj.name, target_objects=[obj], conductor_type="SignalLine", solve_option="SolveOnBoundary", unit="mm"
+)
+
+obj = q.modeler.get_object_from_name("signal_n")
+q.assign_single_conductor(
+    name=obj.name, target_objects=[obj], conductor_type="SignalLine", solve_option="SolveOnBoundary", unit="mm"
+)
+
+# ~~~~~~~~~~~~~~~~~
```

Comment:
@ring630 Maybe we can get ride of it here, but it is up to you.
```suggestion
```

Reply:
Sure. I was copying the coding style from the other example.